integreat_cms/cms/views/pages/page_sbs_view.py

```diff
@@ -71,6 +71,20 @@ def get(self, request, *args, **kwargs):
         source_page_translation = page.get_translation(source_language.slug)
         target_page_translation = page.get_translation(target_language.slug)
 
```

Comment:
```suggestion
        major_target_page_translation = page.translations.filter(
            language__slug=target_language.slug, minor_edit=False
        ).first()

```
I think that we should use the latest major version of the target translation to determine the correct date of the source translation... if I perform a minor edit in the target language (e.g. fix a typo), then this version should not be used to determine the diff of the source translation...
Sure, this is an edge case, but since we had a few problems with edge cases of translation states in the past, I think it would be better to fix it now instead of hoping nobody notices :see_no_evil: